src/pkg/cli/compose.go

```diff
@@ -394,6 +396,7 @@ func createTarball(ctx context.Context, root, dockerfile string) (*bytes.Buffer,
 	gzipWriter := &contextAwareWriter{ctx, gzip.NewWriter(&buf)}
 	tarWriter := tar.NewWriter(gzipWriter)
 
+	doProgress := term.DoColor(term.Stdout) && term.IsTerminal
```

Comment:
Why not use `term.CanColor`? Should we check `!raw` as well like in tail?

Reply:
`CanColor` is only set at startup "can we do color". `DoColor` actually takes the user's choice into account, ie. `NO_COLOR` or `--color=…`... 

The one in `tail.go` is kinda fine, because it's only for the spinner, which is ASCII. In this progress indicator, I use the ANSI "clear line" code :-/